alphastats/gui/utils/ui_helper.py

```diff
@@ -104,20 +120,20 @@ def init_session_state() -> None:
         st.session_state[StateKeys.ORGANISM] = 9606  # human
 
     if StateKeys.WORKFLOW not in st.session_state:
-        st.session_state[StateKeys.WORKFLOW] = [
-            PREPROCESSING_STEPS.REMOVE_CONTAMINATIONS,
-            PREPROCESSING_STEPS.SUBSET,
-            PREPROCESSING_STEPS.REPLACE_ZEROES,
-            PREPROCESSING_STEPS.LOG2_TRANSFORM,
-            PREPROCESSING_STEPS.DROP_UNMEASURED_FEATURES,
-        ]
+        st.session_state[StateKeys.WORKFLOW] = DefaultStates.WORKFLOW
 
     if StateKeys.ANALYSIS_LIST not in st.session_state:
         st.session_state[StateKeys.ANALYSIS_LIST] = []
 
     if StateKeys.LLM_INTEGRATION not in st.session_state:
         st.session_state[StateKeys.LLM_INTEGRATION] = {}
 
+    if StateKeys.ANNOTATION_STORE not in st.session_state:
+        st.session_state[StateKeys.ANNOTATION_STORE] = {}
+
+    if StateKeys.UNIPROT_FIELDS not in st.session_state:
+        st.session_state[StateKeys.UNIPROT_FIELDS] = DefaultStates.UNIPROT_FIELDS
```

Comment:
note that you assigned something mutable to this session state key her, id someone does `st.session_state[StateKeys.UNIPROT_FIELDS].append("123")`, the `DefaultStates.UNIPROT_FIELDS` is altered .. better to a copy here

Reply:
(same above for workflow)